snap/snapcraft.yaml

```diff
@@ -24,40 +25,47 @@ layout:
 apps:
   hotsos:
     command: bin/hotsos
+
 parts:
-  hotsos-cli:
-    plugin: dump
-    source: .
-    override-build: |
-      rm -rf tests
-      git rev-parse --short HEAD > repo-info
-      snapcraftctl build
-    organize:
-      defs: etc/hotsos/defs
```

Comment:
I realized that this wasn't working before because I hadn't copied the `snapcraftctl build` line. But when I re-add it the `organize` section works again as it did before.